apps/previewgenerator.sh

```diff
@@ -48,14 +48,18 @@ else
     # reset the cronjob
     crontab -u www-data -l | grep -v 'preview:pre-generate'  | crontab -u www-data -
     # Remove apps
-    APPS=(ffmpeg php-imagick libmagickcore-6.q16-3-extra)
+    APPS=(php-imagick libmagickcore-6.q16-3-extra)
     for app in "${APPS[@]}"
     do
         if is_this_installed "$app"
         then
             apt purge "$app" -y
         fi
     done
+    if is_this_installed ffmpeg && ! is_app_installed integration_whiteboard
```

Comment:
2. Here you *only* remove it if the Whiteboard app isn't installed.

Why? Doesn't Whiteboard app work without `ffmpeg`?

Reply:
yes, the whiteboard app needs ffmpeg to work correctly, so this is correct👍